index.bs

```diff
@@ -1722,7 +1775,8 @@ WebAuthn supports multiple attestation types:
 
 ### Generating an Attestation Object ### {#generating-an-attestation-object}
 
-This section specifies the algorithm for generating an [=attestation object=] for any [=attestation statement format=].
+This section specifies the algorithm for generating an [=attestation object=] (see: [Figure 3](#fig-attStructs)) for any
```

Comment:
Please proceed to merge then.  If we find better syntax for this, we can always add it later.